devel/ncurses/files/configure.diff

```diff
@@ -1,36 +0,0 @@
---- configure.orig	2015-08-05 19:20:32.000000000 +1000
-+++ configure	2015-08-17 16:48:10.000000000 +1000
-@@ -3166,7 +3166,7 @@
- program_transform_name=`echo $program_transform_name | sed -f conftest.sed`
- rm conftest.sed
- 
--for ac_prog in mawk gawk nawk awk
```

Comment:
I don't think we need to do that. If I was going to always use a specific one it would be the system awk.